lib/ansiblelint/utils.py

```diff
@@ -44,13 +43,15 @@
     from ansible.utils import path_dwim
     from ansible.utils.template import template as ansible_template
     from ansible.utils import module_finder
+
     module_loader = module_finder
     ANSIBLE_VERSION = 1
 except ImportError:
     from ansible.parsing.dataloader import DataLoader
     from ansible.template import Templar
     from ansible.parsing.mod_args import ModuleArgsParser
     from ansible.errors import AnsibleParserError
+
```

Comment:
please don't add non-functional whitespace for code that isn't currently violating PEP8